build/nuget-release.yml

```diff
@@ -0,0 +1,172 @@
+name: $(Package.Version)
+
+trigger: none
+pr: none
+
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      name: arcus-azure/azure-devops-templates
+      endpoint: arcus-azure
+
+variables:
+  - group: 'Build Configuration'
+  # Always use fixed version for .NET Core SDK
+  - name: 'DotNet.Sdk.Version'
+    value: '2.2.402'
+  - name: 'Project'
+    value: 'Arcus.Template'
+  - name: 'Http.Port'
+    value: '5000'
+  - name: 'Arcus_Api_BaseUrl'
+    value: 'http://localhost:$(Http.Port)/api/v1'
+  # 'Package.Version.ManualTrigger' is added as queue-time variable on build in Azure DevOps
+
+stages:
+  - stage: Build
+    jobs:
+      - job: Compile
+        pool:
+          vmImage: 'ubuntu-16.04'
+        steps:
+          - task: DotNetCoreInstaller@0
+            displayName: 'Import .NET Core SDK ($(DotNet.Sdk.Version))'
+            inputs:
+              version: '$(DotNet.Sdk.Version)'
+          - task: DotNetCoreCLI@2
```

Comment:
Looks like we can use this YAML template here, right?

https://github.com/arcus-azure/azure-devops-templates/blob/master/build/build-solution.yml